hack/verify_nvidia_nim_api.go

```diff
@@ -65,12 +67,12 @@ func main() {
 		}
```

Comment:
Why not replace this print with a logger?

Reply:
I thought about that. But I didn't want it to be a part of this PR. The script is just something for us to work with, not consumed by anyone else.
I don't want to block this PR with modifications to this script.

Reply:
@spolti I refactored the hack script, see https://github.com/opendatahub-io/odh-model-controller/pull/392.